docs/volume-health-monitoring.md

```diff
@@ -0,0 +1,30 @@
+# Volume Health Monitoring
+
+The EBS CSI Driver contains experiment support for [Volume Health Monitoring](https://kubernetes.io/docs/concepts/storage/volume-health-monitoring/). Because Volume Health Monitoring is an alpha feature, this support is subject to change as the feature evolves.
+
+## Prerequisites
+
+Volume Health Monitoring requires at least Kubernetes 1.21 or later.
+
+The EBS CSI Driver must be given permission to access the [`DescribeVolumeStatus` EC2 API](https://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_DescribeVolumeStatus.html) for the volumes in the cluster. This example snippet can be used in an IAM policy to grant access to `DescribeVolumeStatus` on all volumes:
+
+```json
+{
+  "Effect": "Allow",
+  "Action": [
+    "ec2:DescribeVolumeStatus"
+  ],
+  "Resource": "*"
```

Comment:
How about restricting to just volumes created by the CSI driver?

Reply:
I think that's probably better left to #1340. Right now, our default example IAM policy gives the driver global permissions for several APIs like `DescribeVolumes`

Reply:
If you're going to add to the entropy that #1340 has to clean up, at least leave a note (if not a new rev) over there.